setup.py

```diff
@@ -55,6 +55,7 @@ def read_file(fname):
 
 install_requires = [
     'colorlog',
+    'CherryPy==6.0.2',
```

Comment:
For our purposes, `install_requires` and `req/base.txt` play the same role, in that they both set up the minimal number of requirements needed to run Kolibri.